autogen.sh

```diff
@@ -952,6 +952,9 @@ if [ -x ./maint/f77tof90 -a $do_f77tof90 = "yes" ] ; then
     echo "done"
 fi
 
+# Invoke test/mpi/autogen.sh to generate dtpools test builds
+pushd test/mpi >& /dev/null && sh ./autogen.sh && popd >& /dev/null
+
```

Comment:
@pavanbalaji this should now be better hopefully. Please note that it is not yet final. The parameter file names should be configurable through env vars and have a default name in case no env var is found. The configurations should also be moved to `test/mpi/dtpools`. One more thing, in order to add new binaries to `testlist` should we have a default `testlist` (with old binaries) that we use to generate to a new testlist by adding new datatype tests? In this case the default `testlist` will be moved to `testlist.in`.